scripts/hudson/narayana.sh

```diff
@@ -985,10 +992,10 @@ export ANT_OPTS="$ANT_OPTS $IPV6_OPTS"
 
 # run the job
 
-[ $NARAYANA_BUILD = 1 ] && build_narayana "$@"
-[ $AS_CLONE = 1 ] && clone_as "$@"
+[ $NARAYANA_BUILD = 1 ] && build_narayana "$@" # NB we should do this after building the AS since it takes longer
+[ $AS_CLONE = 1 ] && AS_BUILD=1 # keep it simple and assume that we are cloning the AS because we want to build it
```

Comment:
I have found a scenario where AS_CLONE=1 and AS_BUILD=0 seems to be useful so I don't think we should prevent that. I am also not spotting any existing PROFILE in https://github.com/jbosstm/narayana/blob/master/scripts/hudson/narayana.sh set as `AS_BUILD=0 AS_CLONE=1` so I don't think it should be needed to changed for this pull request.

Reply:
What was the scenario?

Reply:
To be able to run the tests in the WildFly repo against a downloaded WildFly - that being said to make that useful some further changes in narayana.sh are needed to overwrite the Narayana jars in the downloaded WildFly.

If you want to go this route I think it separate to this pull request. I don't expect that would cause problems for this pull request because I don't think any of the profiles today set AS_CLONE=1 and AS_BUILD=0